laravel/app/ScheduleParser.php

```diff
@@ -178,18 +178,19 @@ private function insertScheduleData($datefile)
     // Notify residents to select new preferences
     public function notifySelectNewPreferences($toName, $toEmail, $date)
     {
-        Log::info("send email");
-        $subject = "REMODEL: Please select new preferences for date " . $date;
-        $body =
-            "Your previous preferences for date " .
-            $date .
+        Log::info("send email for schedule parser ". $date);
+        $subject = config("app.env") == "production" ? "" : "(" .config("app.env") . ") ";
+        $subject .= "REMODEL: Please select new preferences for date " . $date;
+        $body = 
+            "Your previous preferences for date " . 
+            $date . 
             " were deleted because the schedule is updated. Please select new preferences on REMODEL website.";
```

Comment:
This formatting looks strange... I assume it's something prettier did?

Reply:
Yes. I thought it was weird too, do you want me to change it to one line?

Reply:
Yeah. I think Spencer and I fixed something similar with his PR.

If I remember correctly, we fixed it by using string interpolation, rather than concatenation.